driverinference/main.py

```diff
@@ -268,34 +243,22 @@ def inference_colocated(self,
                                                                batch,
                                                                device,
                                                                num_devices,
-                                                               rebuild_model,
                                                                iterations,
                                                                language)
             logger.debug("Inference session created")
+            attach_resnet(infer_session,
+                        f"./imagenet/resnet50.{device}.pt",
+                        device,
+                        num_devices,
+                        batch)
 
             exp.start(infer_session, block=True, summary=True)
 
-            # confirm scaling test run successfully
             stat = exp.get_status(infer_session)
             if stat[0] != status.STATUS_COMPLETED:
                 logger.error(f"One of the scaling tests failed {infer_session.name}")
         self.process_scaling_results(scaling_results_dir=exp_name, plot_type=plot)
 
-
-    @staticmethod
-    def _set_resnet_model(device="GPU", force_rebuild=False):
-            resnet_model = f"./imagenet/resnet50.{device}.pt"
-            if not Path(resnet_model).exists() or force_rebuild:
-                logger.info(f"AI Model {resnet_model} does not exist or rebuild was asked, it will be created")
-                try:
-                    save_model(device)
-                except:
-                    logger.error(f"Could not save {resnet_model} for {device}.")
-                    sys.exit(1)
-
-            logger.info(f"Using model {resnet_model}")
-            return resnet_model
-
```

Comment:
Why did you take this out?

Reply:
bc we are setting the model in the utils file